test/corpus/basic.txt

```diff
@@ -281,3 +281,27 @@ layout (input_attachment_index = 1, set = 1, binding = 1) uniform subpassInput i
           (number_literal))))
     (type_identifier)
     (identifier)))
+
+================================================================================
+Discard
+================================================================================
```

Comment:
You need to run `tree-sitter test -u` because there seems to be another test using `discard`, currently parsed as an expression_statement.